src/web/components/ApiKeyManagement/KeyHelper.spec.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable camelcase */
+import { faker } from '@faker-js/faker';
+
+import { ApiRoleDTO } from '../../../api/entities/ApiRole';
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+import { shouldRotateApiKey, yearInSeconds } from './KeyHelper';
+
+const Mapper: ApiRoleDTO = { id: 1, roleName: 'MAPPER', externalName: 'Mapper', order: 1 };
+const Generator: ApiRoleDTO = {
+  id: 2,
+  roleName: 'GENERATOR',
+  externalName: 'Generator',
+  order: 2,
+};
+const Bidder: ApiRoleDTO = {
+  id: 3,
+  roleName: 'ID_READER',
+  externalName: 'Bidder',
+  order: 4,
+};
+const Sharer: ApiRoleDTO = { id: 4, roleName: 'SHARER', externalName: 'Sharer', order: 3 };
+
+const buildApiKey = (apiKey: Partial<ApiKeyDTO>): ApiKeyDTO => {
```

Comment:
Sorry - we should probably call this `createApiKeyFake()`. We can then export it and use it in the `RotateApiKeyCard` story (unless you wanted to do that in a separate ticket/PR).